src/app-designer-overview.rst

```diff
@@ -12,7 +12,7 @@ The ODK-X Application Designer is accessed through a :program:`Chrome` browser.
 
 This screen has 6 tabs:
 
-  - :ref:`Preview <app-designer-overview-preview>` (shown above) - used to preview `ODK-X Survey <https://docs.odk-x.org/survey-using/>`_ forms and ODK-X Tables list-, detail- and graph- views (future). Displays these within a user-selected device geometry.
+  - :ref:`Preview <app-designer-overview-preview>` (shown above) - used to preview `ODK-X Survey <https://docs.odk-x.org/survey-using/>`_ forms and ODK-X Tables list-views and detail-views (in the future). Displays these within a user-selected device geometry.
```

Comment:
"In the future" is not needed as graph views was the only one in future and was confusing. List and Detail views are both available and are NOT in the future.